MagneticField/ParametrizedEngine/test/testParabolicParametrizedMagneticField.cc

```diff
@@ -0,0 +1,127 @@
+#include <alpaka/alpaka.hpp>
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/FileInPath.h"
+#include "DataFormats/GeometryVector/interface/GlobalPoint.h"
+#include "DataFormats/GeometryVector/interface/GlobalVector.h"
+#include "DataFormats/FWLite/interface/Record.h"
+#include "FWCore/Utilities/interface/Exception.h"
+
+#include "MagneticField/ParametrizedEngine/interface/alpaka/ParabolicParametrizedMagneticField.h"
+
+#include "FWCore/Utilities/interface/stringize.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/memory.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/workdivision.h"
+
+#include <iostream>
+#include <fstream>
+#include <Eigen/Core>
+
+using namespace edm;
+using namespace std;
+using namespace alpaka;
+using namespace ALPAKA_ACCELERATOR_NAMESPACE;
+using namespace ALPAKA_ACCELERATOR_NAMESPACE::magneticFieldParabolicPortable;
+using Vector3f = Eigen::Matrix<float, 3, 1>;
+
+struct MagneticFieldKernel {
+  template <typename TAcc, typename T>
+  ALPAKA_FN_ACC void operator()(
+      TAcc const& acc, T const* __restrict__ in, T* __restrict__ out, size_t size) const {
+    for (auto index : cms::alpakatools::uniform_elements(acc, size)) {
+      out[index][0] = 0;
+      out[index][1] = 0;
+      out[index][2] = magneticFieldAtPoint(in[index]);
+    }
+  }
+};
+
+int main() {
```

Comment:
Can you add a check and fail with an error is there are no devices, similar to the other alpaka tests ?
```c++
  // get the list of devices on the current platform
  auto const& devices = cms::alpakatools::devices<Platform>();
  if (devices.empty()) {
    std::cerr << "No devices available for the " EDM_STRINGIZE(ALPAKA_ACCELERATOR_NAMESPACE) " backend, "
      "the test will be skipped.\n";
    exit(EXIT_FAILURE);
  }
```

Reply:
OK: https://github.com/cms-sw/cmssw/pull/44360/commits/8d93f61cc051bd4b2a39d6559170dd00f6a2bcef